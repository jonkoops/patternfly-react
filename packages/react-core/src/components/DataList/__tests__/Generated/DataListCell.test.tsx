/**
 * @vitest-environment jsdom
 */
import { render } from '@testing-library/react';
import React from 'react';
import { expect, it } from 'vitest';

import { DataListCell } from '../../DataListCell';

it('DataListCell should match snapshot (auto-generated)', () => {
  const { asFragment } = render(
    <DataListCell
      children={<>ReactNode</>}
      className={"''"}
      width={1}
      isFilled={true}
      alignRight={false}
      isIcon={false}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
