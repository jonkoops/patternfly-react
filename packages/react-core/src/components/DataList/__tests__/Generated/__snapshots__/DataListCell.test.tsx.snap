// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DataListCell should match snapshot (auto-generated) 1`] = `
<DocumentFragment>
  <div
    class="pf-v5-c-data-list__cell ''"
  >
    ReactNode
  </div>
</DocumentFragment>
`;
